identity-account/src/account/account.rs

```diff
@@ -471,6 +472,29 @@ impl Account {
     Ok(())
   }
 
+  /// Fetches the latest changes for the account did from the tangle and
+  /// updates the local state
+  pub async fn synchronize_state(&mut self) -> Result<()> {
```

Comment:
Should we add the force flag in case the user wants to save the incoming changes in the storage?